data/json/npcs/talk_tags.json

```diff
@@ -1320,7 +1744,10 @@
       "Somebody get some water!",
       "Fire, fire, FIRE!",
       "Get an extinguisher!",
-      "Danger hot!"
+      "Danger hot!",
```

Comment:
`"Danger!  Hot!",`

Reply:
I think this is meant to be a play on the term 'Danger Close', And if that's what it is, I think it can stay.